microservices/coco-attestation/uyuni-coco-attestation.spec

```
@@ -0,0 +1,113 @@
+#
+# spec file for package coco-attestation
+#
+# Copyright (c) 2024 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+Name:           uyuni-coco-attestation
+Version:        1.0.0
```

Like for the other spec files  in this  PR,  let's align with the other  sources of SUSE Manager 5.0

```suggestion
Version:        5.0.0
```

you can also go with a beta release  or something similar,  but  please, let's  use  a 5.X